src/System.Net.Http.WinHttpHandler/tests/UnitTests/FakeInterop.cs

```diff
@@ -388,6 +388,15 @@ private static bool CopyToBufferOrFailIfInsufficientBufferLength(string value, I
             return true;
         }
 
+        public static bool WinHttpQueryOption(
+            SafeWinHttpHandle handle,
+            uint option,
+            ref uint buffer,
+            ref uint bufferSize)
+        {
+            return true;
```

Comment:
ok.  In a world where the unit tests would actually be testing for that specific option, yes, it might be needed. But I don't have any unit tests that require that. So, getting the unit tests to compile and run with zero errors is all that is needed.

If, however, we added new unit tests that needed more "real" functionality to work, then it would be required.